test/Common/ClientFunctions.cs

```diff
@@ -25,5 +26,23 @@ internal static class ClientFunctions
                 instanceId,
                 instanceCreationTime);
         }
+
+        [NoAutomaticTrigger]
+        public static async Task StartFunctionWithTaskHub(
+            [OrchestrationClient(TaskHub = "testtaskhubV2")] DurableOrchestrationClient client,
```

Comment:
If you want to specify a dynamic value for the task hub name, consider binding it to some input data or using a `Binder` object to do it imperatively in the code.

Reply:
Great!

I will work on it!

Thank you!

Reply:
Hello @cgillum,

I know that you are super busy with the release but if you have time it will be great to hear your opinion.

Do you know if there a sample I can check about passing dynamic value to the  `OrchestrtrationClientAttribute`?

I tried adding the `%{value}%` notation for the `TaskHub` in the `OrchestrtrationClientAttribute` after setting the mocked `INameResolver` as Yuka did but I don't think this approach works well with `AutoResolve` attribute. I am getting the following error when I use the `%{value}%` notation for the `TaskHub` in the `OrchestrtrationClientAttribute` :

**"Message: Microsoft.Azure.WebJobs.Host.Indexers.FunctionIndexingException : Error indexing method 'TestOrchestrations.SayHelloInlineTaskHubName'
---- System.InvalidOperationException : '%TaskHubName%' does not resolve to a value.
"**

Thank you!

Reply:
Hello @cgillum,

Thank you very much!

I will try it and let you know.

My approach was to use moq to mock the interface.

Thank you!